cli/cli.go

```diff
@@ -344,17 +344,15 @@ OPTIONS:
 								"logs from " +
 								"ERROR level",
 						},
+						cli.StringFlag{
```

Comment:
Hmm... I would propose moving all these flags up one level to the main command scope as they will probably become relevant for the upcoming "upload-to-server" sub-command.